### PR TITLE
Pop basics update

### PR DESCRIPTION
**Pull request by storyaddict**
# This PR Does the following:

- Updates screenshots and reorganizes content for better flow.
- Includes in-line video tutorials for pop-shell features.
- Adds screenshots of changes to 21.04 and 21.10 desktop menus.
- Cross references with other articles and Github pages for convenience.

## Notes:
I'm open to suggestions if there is a better way to organize the sections of the article. I went back and forth on the current set-up.

### CODE REVIEW

File: content/pop-basics.md [hunk 1]
```diff
@@ -17,30 +17,169 @@ section: pop
 tableOfContents: true
 ---
 
+Welcome to the Pop!_OS desktop! By default, it's clean and ready for action.
 The Pop!_OS operating system provides all the features you would expect from a modern desktop.
 
-## Getting to Know The Pop!_OS Desktop
+Pop!\_OS is based on Ubuntu, however, extra features and tools have been added, while other aspects have been slimmed down. You can read more about Pop!\_OS features and the differences between Pop!\_OS and Ubuntu [here](/articles/difference-between-pop-ubuntu)
```
Comment (ahoneybun) [inline content/pop-basics.md:23]: Missing a period.

File: content/pop-basics.md [hunk 2]
```diff
@@ -17,30 +17,169 @@ section: pop
...
 
-![Pop Desktop](/images/pop-basics/pop-desktop.png)
+## Pop!_OS Desktop Environment
 
-Welcome to the Pop!_OS desktop! By default, it's clean and ready for action.
+The Pop!_OS Desktop environment is based on GNOME shell and layered with other components called **Pop!\_Shell** and **C.O.S.M.I.C. (Computer Operating System Main Interface Components)** For more info, including source-code, refer to [this page](https://github.com/pop-os/).
```
Comment (ahoneybun) [inline content/pop-basics.md:27]: Isn't Source Code two words?

File: content/pop-basics.md [hunk 3]
```diff
@@ -17,30 +17,169 @@ section: pop
...
-To navigate within the desktop, either click the <u>Activities</u> button in the top left, or press the <kbd><font-awesome-icon :icon="['fab', 'pop-os']"></font-awesome-icon></kbd> key on the keyboard.  This will show an overview of all open windows and provide a text box to search your system.
+1. Click on the Workspaces icon in the Dock.
+2. Click on `Workspaces` in the top-left.
+3. Swipe left on the trackpad with four (4) fingers.
+4. Press <kbd>SUPER</kbd> + <kbd>D</kbd>
+
+Once in the `Workspaces` overview, you can move active application windows to different workspaces by clicking and dragging them, or by pressing - <kbd>SUPER</kbd> + <kbd>SHIFT</kbd> + <kbd>↑</kbd> / <kbd>↓</kbd>.
+
+Click on the thumbnail for a specific workspace to switch to it, or use
+<kbd>SUPER</kbd> + <kbd>CTRL</kbd> + <kbd>↑</kbd> / <kbd>↓</kbd> to switch to workspaces above or below the current one.
```
Comment (ahoneybun) [inline content/pop-basics.md:113]: I think this should be Super+Enter then press these arrow keys to move windows.

File: content/pop-basics.md [hunk 4]
```diff
@@ -17,30 +17,169 @@ section: pop
...
+
+**NOTE:** Current versions of Pop!\_Shell also offer options for showing or hiding window title bars.
+
+### Keyboard Shortcuts
+
+<iframe width="900" height="500" src="https://www.youtube.com/embed/aqj0cRTZaVE" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+This video covers keyboard shortcuts for opening the `Launcher`, opening applications, and switching between open applications or workspaces. It also covers how to navigate and adjust open applications windows.
+
+For a more detailed overview of Pop!\_OS keyboard shortcuts, refer to [this article](/articles/pop-keyboard-shortcuts)
```
Comment (ahoneybun) [inline content/pop-basics.md:91]: Missing a period.

File: content/pop-basics.md [hunk 5]
```diff
@@ -17,30 +17,169 @@ section: pop
...
+
+Pop!\_OS 21.10 introduced a new UI for the `Applications` menu.
+
+By default, all apps installed are shown in alphabetical order. Application folders are grouped along the bottom of the window. New folders can be created, and apps can be dragged into, or out of folders.
+
+If a folder is deleted, all applications in the folder revert to their original location and alphabetical order in the main menu.
+
+![new-app-menu](/images/pop-basics/21-10-apps-menu.png)
+
+As in previous versions a search box is provided which can search installed applications, or applications to install from the [Pop!\_Shop](#pop_shop-app-installation)
```
Comment (ahoneybun) [inline content/pop-basics.md:139]: Missing a period.

File: content/pop-basics.md [hunk 6]
```diff
@@ -49,6 +188,14 @@ For many applications, additional options are available in a separate menu withi
...
 
 ![Top right menu](/images/pop-basics/pop-settings.png)
+
+## Pop!_Shop (App Installation)
+
+The <u>Pop!_Shop</u> can be used to install additional software. Just search for programs or browse for them by category, and click the `Install` button to add them to your computer.
+
+![Pop!_Shop](/images/pop-basics/pop-shop.png)
+
+To learn more about the Pop!_Shop refer to [this page](https://github.com/pop-os/shop), and for more information about package management in Pop!\_OS see this [article](articles/manage-repos-pop)
```
Comment (ahoneybun) [inline content/pop-basics.md:201]: Missing a period.

**Review by ahoneybun — CHANGES_REQUESTED** [review]

File: content/pop-basics.md (hunk 1 above)
Comment (storyaddict) [reply content/pop-basics.md:23]: Added. This was a previously recommended style choice. I'm still trying to break that habit.

File: content/pop-basics.md (hunk 2 above)
Comment (storyaddict) [reply content/pop-basics.md:27]: I think it could go either way, but I'll do "source code" instead.

File: content/pop-basics.md (hunk 4 above)
Comment (storyaddict) [reply content/pop-basics.md:91]: Added.

File: content/pop-basics.md (hunk 3 above)
Comment (storyaddict) [reply content/pop-basics.md:113]: Fixed.

File: content/pop-basics.md (hunk 5 above)
Comment (storyaddict) [reply content/pop-basics.md:139]: Fixed.

File: content/pop-basics.md (hunk 6 above)
Comment (storyaddict) [reply content/pop-basics.md:201]: Fixed.

**Review by storyaddict — COMMENTED** [review]
Made Aaron's suggested changes.

**Review by ahoneybun — APPROVED** [review]

**Pull request merged**